modules/twinklewarn.js

```diff
@@ -1171,6 +1177,17 @@ Twinkle.warn.callback.change_category = function twinklewarnCallbackChangeCatego
 			// create the options
 			createEntries(groupContents, optgroup, false);
 		});
+	} else { // All of the above
+		['level1', 'level2', 'level3', 'level4', 'level4im'].forEach(function(lvl) {
+			value = lvl;
```

Comment:
This is not the way to do it. Tweak the `createEntries` function to take the level as an argument. Removes the need for the `value = '';` kludge below. Using arguments is better than using closures.

Reply:
Quite right, off course.  I think now that was my intent back in June, but did this to get something working quick and just totally skipped over that memory five months later.